src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -103,6 +135,31 @@ class TextSentimentSuiteV4 extends TestBase with DataFrameEquality with TextKey
       data(0).get(2).toString == "negative")
     assert(data(1).get(0).toString == "positive")
   }
+
+  lazy val df2: DataFrame = Seq(
+    ("en", "Hello world. This is some input text that I love."),
+    ("fr", "Bonjour tout le monde"),
+    ("es", "La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+    (null, "ich bin ein berliner")
+  ).toDF("lang", "text")
+
+  test("Sentiment - batch usage"){
+    lazy val detector3: TextSentimentV4  = new TextSentimentV4(options)
+      .setSubscriptionKey(textKey)
+      .setEndpoint("https://eastus.api.cognitive.microsoft.com/")
+      .setInputCol("text")
+      .setLangCol("lang")
+      .setOutputCol("output2")
+
+    val batchedDF2 = new FixedMiniBatchTransformer().setBatchSize(10).transform(df2.coalesce(1))

```

Comment:
I'm confused here. Shouldn't the change be in the TextSentimentV4 class, and not outside of it.
Looks like this PR just show-case how a customer might use the `FixedMiniBatchTransformer` to do batching but doesn't change the connector to do it instead.

Shouldn't `TextAnalyticsSDKBase` inherit or use `FixedMiniBatchTransformer` somehow instead?